### ISSUE TITLE
Organize Imports should never result in all imports being deleted

### ISSUE THREAD

**Issue by anthonyvdotbe** [redhat-developer/vscode-java#1383]
Sometimes, in case of a syntax error such as a missing semi-colon, when doing `Organize Imports`, all imports are deleted (see also #430 e.g.). This is rarely (if ever) a good choice. If, for whatever reason, `Organize Imports` is unable to determine the correct set of imports, it should just do nothing at all.

##### Environment
- Operating System: Windows 10
- JDK version: JDK 14
- Visual Studio Code version: 1.44.0
- Java extension version: 0.59.1

##### Steps To Reproduce
1. create a class App.java as follows
2. do `Organize Imports`

````java
import java.net.URI;
import java.net.http.HttpClient;
import java.net.http.HttpClient.Redirect;
import java.net.http.HttpRequest;

import javax.xml.parsers.DocumentBuilder;

class App {

    public static void main(String... args) {
        HttpClient client = HttpClient.newBuilder().followRedirects(Redirect.NORMAL).build();
        DocumentBuilder builder = null;
        
        URI season = URI.create("foo");
        client.send(HttpRequest.newBuilder(null), null)
    }

}
````

##### Current Result
All imports are deleted.

##### Expected Result
The imports are untouched.

**Comment by fbricon** [comment]
Agreed, this is a jdt.ls bug, since the problem can not be reproduced in Eclipse.

**Comment by fbricon** [comment]
Looks like this is a combination of malformed code and maybe specific statements, as I can't reproduce this issue in other cases, with a simple missing  semicolon or parenthesis.

**Comment by snjeza** [comment]
I can reproduce the issue using VS Code 0.59.1, but it is fixed in the 0.66.0 version.
